charts/cray-certmanager/templates/cert-manager/rbac.yaml

```diff
@@ -0,0 +1,31 @@
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cert-manager
```

Comment:
The names of these objects should be more descriptive, for example `cert-manager-hook`